src/pkg/selectors/sharepoint.go

```diff
@@ -640,7 +646,7 @@ func (s SharePointScope) set(cat sharePointCategory, v []string, opts ...option)
 	os := []option{}
 
 	switch cat {
-	case SharePointLibraryFolder, SharePointList, SharePointPage:
+	case SharePointLibraryFolder, SharePointPage:
```

Comment:
Could you please explain why list was removed from here?

Reply:
@pandeyabs 
for lists, 
- there is no nested folders -> there cannot be lists within other lists
- list itself is the item -> so container and item are the same

so, since there is no path involved here, thought to remove it from here as we do not need any `path` filters. Plus wanted to enforce `StrictEqual` check to select lists provided using `--list` flag.

Reply:
Thanks for clarifying! Could you please add this note in this function for future reference?

Reply:
@pandeyabs Added